### PR TITLE
 refs #5 - [DO NOT MERGE] adding travis ci & code sniffer

### PR DESCRIPTION
**Pull request by mbrenguier**

### CODE REVIEW

File: Library/Services/ManagerServiceInterface.php [hunk 1]
```diff
@@ -1,6 +1,32 @@
 <?php
+/**
+ * ManagerServiceInterface
+ *
+ * Provides helper functions that act upon a token array and modify the file
```
Comment (jrmbrgs) [inline Library/Services/ManagerServiceInterface.php:5]: ?? :p

File: Library/Services/ManagerServiceInterface.php [hunk 2]
```diff
@@ -1,6 +1,32 @@
...
+ * content.
+ *
+ * @category Test
+ * @package  Disturb
+ * @author   Maxime BRENGUIER <mbrenguier@voyageprive.com>
+ * @license  https://github.com/vpg/disturb/blob/poc/LICENSE MIT Licence
+ * @version  0.0.1
```
Comment (jrmbrgs) [inline Library/Services/ManagerServiceInterface.php:12]: ??

File: Library/Services/ManagerServiceInterface.php [hunk 3]
```diff
@@ -1,6 +1,32 @@
 <?php
+/**
+ * ManagerServiceInterface
+ *
+ * Provides helper functions that act upon a token array and modify the file
+ * content.
+ *
+ * @category Test
```
Comment (jrmbrgs) [inline Library/Services/ManagerServiceInterface.php:8]: ??

File: Library/Services/ManagerServiceInterface.php [hunk 4]
```diff
@@ -1,6 +1,32 @@
...
+ * @package  Disturb
+ * @author   Maxime BRENGUIER <mbrenguier@voyageprive.com>
+ * @license  https://github.com/vpg/disturb/blob/poc/LICENSE MIT Licence
+ * @version  0.0.1
+ * @link     http://example.com/my/bar Documentation of Foo.
```
Comment (jrmbrgs) [inline Library/Services/ManagerServiceInterface.php:13]: rm

File: Library/Services/WorkflowManager.php [hunk 5]
```diff
@@ -26,47 +27,262 @@ public function __construct(string $workflowConfigFilePath)
...
         $this->getDI()->get('logger')->debug(json_encode(func_get_args()));
         $nextStepPos = $this->tmpStorage[$workflowProcessId]['currentStepPos'] + 1;
-        $this->getDI()->get('logger')->debug($nextStepPos);
-        $stepHash = $this->config->steps[$nextStepPos]->toArray();
-        // Deals w/ parallelized task xxx to unitest
-        if (array_keys($stepHash) !== array_keys(array_keys($stepHash))) {
-            return [$stepHash];
+
+        // Manage case when there is no more step to run
+        if(empty($this->config->steps[$nextStepPos])) {
```
Comment (jrmbrgs) [inline Library/Services/WorkflowManager.php:99]: this has already been moved in a dedicated func..
This PR sounds not up to date

File: Library/Services/WorkflowManager.php [hunk 6]
```diff
@@ -26,47 +27,262 @@ public function __construct(string $workflowConfigFilePath)
...
     }
 
-    public function finalizeStep(string $workflowProcessId, string $stepCode, array $resultHash) {
+    /**
+     * Check current step status and if we can go further in the workflow
+     *
+     * @param string $workflowProcessId
+     * @return bool
+     */
+    public function isNextStepRunnable(string $workflowProcessId) : bool
```
Comment (jrmbrgs) [inline Library/Services/WorkflowManager.php:136]: obsolete code, replaced by getCurrentStepStatus.
Please rebase

**Review by jrmbrgs — CHANGES_REQUESTED** [review]
maybe it's a TEST PR, to validate the CI setup...
If not, it sounds there are too many changed file not related to the PR topic.

**Comment by mbrenguier** [comment]
this PR helps me to create "test" builds + code sniffer behavior

**Pull request closed without merging**